ext/bigdecimal/bigdecimal.c

```diff
@@ -3500,6 +3500,9 @@ rb_convert_to_BigDecimal(VALUE val, size_t digs, int raise_exception)
  *  in the value, the result is rounded to that number of digits,
  *  according to the current rounding mode; see BigDecimal.mode.
  *
+ *  When +ndigits+ is 0, the number of digits to correctly represent a float number
+ *  is determined by a +dtoa+ algorithm (see more in C function +dtoa+ with +mode=0+).
```

Comment:
Could you please remove the mention of `dtoa`.  This is not the specification but just the current implementation detail.

Reply:
done